data/authors.yml

```diff
@@ -2849,3 +2849,7 @@ amarchuk:
 igoihman:
   name: Irit Goihman
   gravatar: d12f3a20618855ef37a17dab9379dbe5
+
+sabusaale:
```

Comment:
I think there's a typo with the double a in the username

Reply:
Fixed, thanks:)